clloader/loader.py

```diff
@@ -121,22 +54,33 @@ def __init__(
     def _define_increments(self, increment: Union[List[int], int],
                            initial_increment: int) -> List[int]:
         if isinstance(increment, list):
-            return increment
-        increments = []
-        if initial_increment:
-            increments.append(initial_increment)
-
-        nb_tasks = (self.nb_classes - initial_increment) / increment
-        if not nb_tasks.is_integer():
-            raise Exception(
-                "The tasks won't have an equal number of classes"
-                f" with {len(self.class_order)} and increment {increment}"
-            )
-        increments.extend([increment for _ in range(int(nb_tasks))])
+
+            # Check if the total number of classes is compatible between increment list and self.nb_classes
+            if not (np.array(increment).sum()== self.nb_classes()):
```

Comment:
`if not sum(increment) == self.nb_classes` is simpler. The builtin `sum` does the trick, and because `self.nb_classes` has a property decorator, it doesn't need to be explicitly called.